eng/dotnet-format/dotnet-format-integration.yml

```diff
@@ -0,0 +1,90 @@
+jobs:
+- job: Formatting_Check
+  pool:
+    vmImage: 'ubuntu-latest'
+  timeoutInMinutes: 10
+  steps:
+    - script: ./restore.sh && echo "##vso[task.setvariable variable=PATH]$(Build.SourcesDirectory)/.dotnet:${PATH}" && echo "##vso[task.setvariable variable=DOTNET_ROOT]$(Build.SourcesDirectory)/.dotnet"
+      displayName: Restore dependencies
+    - script: dotnet run --project ./src/BuiltInTools/dotnet-format/dotnet-format.csproj -c Release -- @eng/dotnet-format/validate.rsp
+      displayName: Run dotnet-format
+    - task: PublishBuildArtifacts@1
+      displayName: Publish Logs
+      inputs:
+        PathtoPublish: '$(Build.SourcesDirectory)/artifacts/log/'
+        ArtifactName: 'dotnet-format formatting Check'
+      continueOnError: true
+      condition: not(succeeded())
+
+- job: dotnet_format_integration_tests
+  pool:
+    vmImage: 'windows-latest'
+  strategy:
+    maxParallel: 8
+    matrix:
+      format:
+        _repo: "https://github.com/dotnet/format"
```

Comment:
May want to revisit this list.

Reply:
I just delete this, since we already run format on itself